jfix-armeria-aggregating-profiler/src/main/kotlin/ru/fix/armeria/aggregating/profiler/ProfiledError.kt

```diff
@@ -63,12 +62,12 @@ internal sealed class ProfiledError(
     }
 }
 
-internal fun RequestLog.detectProfiledErrorIfAny(): ProfiledError? {
+internal fun RequestLog.detectProfiledErrorIfAny(isResponseStatusValid: (HttpStatus) -> Boolean): ProfiledError? {
     val requestCause = requestCause()
     val responseCause = responseCause()
     val status = responseHeaders().status()
     return when {
-        status.codeClass() != HttpStatusClass.SUCCESS && status != HttpStatus.UNKNOWN -> ProfiledError.InvalidStatus
+        !isResponseStatusValid(status) -> ProfiledError.InvalidStatus
```

Comment:
UNKNOWN is a special value representing that http status is not available (e.g. some request error occured). I think it is a bad idea to hope that lib clients will not forget considering this special value

Reply:
done